cloud-haven-react-app/src/App.css

```diff
@@ -19,5 +19,5 @@
    background-color: white;
    border-top-left-radius: 20px;
    display: block;
-   overflow: auto;
+   overflow: hidden;
```

Comment:
The reason I put `auto` here originally is so that users will be able to scroll to view any parts of the page that go lower. Hidden will mean that the user won't be able to see any data that doesn't fit inside the frame